src/BridgedRoom.ts

```diff
@@ -594,7 +594,7 @@ export class BridgedRoom {
         this.addRecentSlackMessage(message.ts);
         try {
             const ghost = await this.main.ghostStore.getForSlackMessage(message, this.slackTeamId!);
-            await ghost.update(message, this);
+            await ghost.update(message, this, this.main.config);
```

Comment:
There probably isn't much point passing in `this`, and also `this.main.config`

Reply:
`main` is private, so we can't access `this.main.config`

Reply:
Ah, how annoying. Fair doos.